src/backend/main.ts

```diff
@@ -638,18 +638,11 @@ ipcMain.handle(
 
 /// IPC handlers begin here.
 
-ipcMain.handle('checkGameUpdates', async (event, library?: Runner) => {
-  switch (library) {
-    case 'legendary':
-      return LegendaryLibrary.get().listUpdateableGames()
-    case 'gog':
-      return GOGLibrary.get().listUpdateableGames()
-    default:
-      return [
-        ...(await LegendaryLibrary.get().listUpdateableGames()),
-        ...(await GOGLibrary.get().listUpdateableGames())
-      ]
-  }
+ipcMain.handle('checkGameUpdates', async () => {
+  return [
+    ...(await LegendaryLibrary.get().listUpdateableGames()),
```

Comment:
Is here catched that a user only uses one store?

Reply:
For Legendary, if you're not logged in, `listUpdateableGames()` will just return `[]`. GOG does a similar thing, looping through the installed games and checking for an update one-by-one

Having GOG games installed, logging out of GOG & then refreshing the library might lead to issues, although that should be easy to fix.  
I'm not quite sure though, maybe @imLinguin can confirm this